pkg/node/node.go

```diff
@@ -309,20 +294,12 @@ func (n *Node) Marshal() ([]byte, error) {
 
 // Unmarshal parses the JSON byte slice and updates the node receiver
 func (n *Node) Unmarshal(data []byte) error {
-	// Backup private fields
-	cluster := n.cluster
-	dev := n.dev
-
 	newNode := Node{}
 	if err := json.Unmarshal(data, &newNode); err != nil {
 		return err
 	}
 
 	*n = newNode
```

Comment:
This still needs to be solved to fix https://github.com/cilium/cilium/issues/6572

Reply:
ACK. I originally had the fix in but opted to keep it outside so we can push a fix that we can also backport to 1.3